ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/InternalPaywall.kt

```diff
@@ -125,6 +127,7 @@ private fun LoadedPaywall(state: PaywallState.Loaded, viewModel: PaywallViewMode
         val configuration = state.configurationWithOverriddenLocale()
 
         CompositionLocalProvider(
+            LocalActivity provides LocalContext.current.getActivity(),
```

Comment:
This shouldn't cause any leaks since any changes to the `LocalContext.current` would cause a recomposition to the composition local provider